portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Portlet.path

```diff
@@ -18,6 +18,26 @@
 	<td>//div[@id='main-content']//section[contains(.,'${key_portletName}')]//div[@class='portlet-body']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>BORDER_STYLE_COLOR_INPUT</td>
+	<td>//div[contains(@class,'border-color')]//label[contains(.,'${key_fieldLabel}')]/following-sibling::div/div/input</td>
+	<td></td>
+</tr>
+<tr>
+	<td>BORDER_STYLE_SELECT</td>
+	<td>//div[contains(@class,'border-style')]//select[contains(@id,'${key_fieldLabel}')] </td>
+	<td></td>
+</tr>
+<tr>
+	<td>BORDER_STYLE_WIDTH_INPUT</td>
+	<td>//div[contains(@class,'border-width')]//input[contains(@id,'${key_fieldLabel}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>BORDER_STYLE_WIDTH_SELECT</td>
+	<td>//div[contains(@class,'border-width')]//select[contains(@id,'${key_fieldLabel}')]</td>
+	<td></td>
+</tr>
```

Comment:
Move to Select.path file then rename a general one
ANY_ID
//select[contains(@id,'${key_value}')]